zulip/integrations/bridge_with_slack/README.md

```diff
@@ -5,30 +5,42 @@ This is a bridge between Slack and Zulip.
 ## Usage
 
 ### 1. Zulip endpoint
+
 1. Create a generic Zulip bot, with a full name like `Slack Bot`.
-2. (Important) Subscribe the bot user to the Zulip stream you'd like to bridge your Slack
-   channel into.
-3. In the `zulip` section of the configuration file, enter the bot's `zuliprc`
+
+2. (Important) Subscribe the bot user to the Zulip channel you'd like to bridge
+   your Slack channel into.
+
+3. Create a [Slack Webhook integration bot](https://zulip.com/integrations/doc/slack)
+   to get messages form Slack to Zulip.
+
+4. In the `zulip` section of the configuration file, fill `integration_bot_email`
+   with **Integration bot**'s email. Note that this is the bot you created in
```

Comment:
Can't we use the same Zulip bot user for both parts?

Reply:
Ahh, that'd be neat. Ok

Reply:
hmm, it might be nicer if we add the instruction to use a generic bot instead of an incoming webhook bot if the user is setting up a Slack bridge in the Slack webhook integration doc instead of here.

Opened [ #32714](https://github.com/zulip/zulip/pull/32714) for this